universal-application-tool-0.0.1/conf/application.conf

```diff
@@ -229,7 +230,7 @@ play.filters {
   # Play then verifies that both tokens are present and match.
   csrf {
     # Sets the cookie to be sent only over HTTPS
-    #cookie.secure = true
+    cookie.secure = false
```

Comment:
suggestion (non-blocking): set this to true in non-local and test envs or file an issue to do so